api/src/routing.js

```diff
@@ -655,6 +655,14 @@ app.all(
   dbDocHandler.request,
   authorization.setAuthorized // adds the `authorized` flag to the `req` object, so it passes the firewall
 );
+const initialReplication = require('./controllers/initial-replication');
```

Comment:
Put the `require` at the top of the doc so it fails fast if the file is missing.